src/test/shared/codelens/sampleGoSamProgram.ts

```diff
@@ -0,0 +1,145 @@
+/*!
```

Comment:
nit: It seems these existing ones are tests only, any reason this isn't combined with `goCodelensProvider`?

Reply:
The C# CodeLens tests have the same structure, which I think makes more sense than combining them.

Reply:
There's no reason to split files if it won't be reused, it's just another thing to have to find when you want to look at how it works